incubator/nfs-pv-provider/templates/nfs-server-deploy.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: nfs-server
+  labels:
+    app: {{ template "nfs-pv-provider.name" . }}
+    chart: {{ template "nfs-pv-provider.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    role: nfs-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "nfs-pv-provider.name" . }}
+      chart: {{ template "nfs-pv-provider.chart" . }}
```

Comment:
Label selectors shold have the `release` label, not the `chart` label. Apply everywhere.

Reply:
I think everywhere is just both places, here in the deployment... thanks!

2aac6782